app/components/Account/CollateralTable.jsx

```diff
@@ -0,0 +1,255 @@
+import React from "react";
+import MarginPosition from "./MarginPosition";
+import ChainTypes from "../Utility/ChainTypes";
+import BindToChainState from "../Utility/BindToChainState";
+import AssetWrapper from "../Utility/AssetWrapper";
+import {ChainStore} from "bitsharesjs";
+import Translate from "react-translate-component";
+import counterpart from "counterpart";
+import {List} from "immutable";
+import TranslateWithLinks from "../Utility/TranslateWithLinks";
+import Immutable from "immutable";
+
+const alignRight = {textAlign: "right"};
+const alignLeft = {textAlign: "left"};
+
+/**
+ * Renders whole margin list including call orders and placeholders
+ */
+class ListGenerator extends React.Component {
+    static propTypes = {
+        callOrders: ChainTypes.ChainObjectsList
+    };
+
+    constructor(props) {
+        super(props);
+
+        this.state = {
+            assetsPropsCount: 0,
+            callOrdersCount: 0
```

Comment:
this is unused?

Reply:
why not? used
at the bottom of the exported component.

Reply:
Can't find it, maybe not commited?

![image](https://user-images.githubusercontent.com/33128181/48474734-7923d880-e7fb-11e8-928d-fecd7b623ed9.png)

Reply:
i meant "yes" - unused. removed